assets/src/block-templates/main-theme-url.js

```diff
@@ -0,0 +1 @@
+export default `${window.p4bk_vars.themeUrl || ''}`;
```

Comment:
Do we need to use template literals here? Since`themeUrl` it's always an `string` https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/e312e0d4d1c6c00acf6a0955932d8d5cf2ef6223/classes/class-loader.php#L404
I think `export default window.p4bk_vars.themeUrl || ''` it's fine.  Am I correct?